### DISCUSSION TITLE
im confused

### DISCUSSION THREAD

**Discussion by powderblaster** [ErinaSugino/Starbound-SBR-Mod#77] [Trouble Shooting] [answered]
i have been trying to get the mod working but it dosent seem to want to
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/13703212/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
`[21:35:25.188] [Warn] Root: Overriding duplicate asset source '..\mods\SxB_-_TemTems_Teratoscenes.pak' named 'sexbound_TeratophiliaScenesPack' with higher or equal priority source '..\mods\TeratophiliaScenes`

You have two version of TeratophiliaScenes installed, and the older one is overriding the newer one.

**Reply by powderblaster** [reply]
ill try removeing one

**Reply by ErinaSugino** [reply]
Not just "one", the wrong one. The one **without** "SxB_" at the beginning.

**Reply by powderblaster** [reply]
yea i just removed the folder one and its fixed

**Reply by powderblaster** [reply]
i have been trying for almost a month before i noticed this page
